webapp/managers.py

```diff
@@ -324,6 +324,29 @@ def __get_statuses(self, group: int) -> dict[tuple[int, int], TaskStatus]:
         return dictionary
 
 
+class HomeManager:
+    def __init__(
+        self,
+        statuses: StatusManager
+    ):
+        self.statuses = statuses
+
+    def get_group_place(self, gid: int) -> int:
+        groupings = self.statuses.get_group_rating()
+        for place in groupings.keys():
```

Comment:
```suggestion
        for place in groupings:
```
`dict` сам по себе поддерживает итерацию по ключам. А ещё можно сразу пройти и по ключам, и по значениям:
```suggestion
        for place, groups_in_place in groupings.items():
```
Функцию ниже тоже можно упростить.

Reply:
Спасибо за замечание! Упростил выражение в [fix: optimized HomeManager](https://github.com/kispython-ru/dta/pull/56/commits/036f629b3a31603196e249b3fe14ff77a7a3d328)